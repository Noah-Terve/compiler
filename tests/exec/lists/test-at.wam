list @l int @r a = [1, 2, 3, 4];
printlnlist @l int @r (a);
/*
[1, 2, 3, 4];
println @l int @r (list_at @l int @r (a, 0));
    */


println @l string @r ("abc"); 

list @l string @r b = ["1", "2", "3", "4"];
println @l string @r (list_at @l string @r (b, 0));
println @l string @r (list_at @l string @r (b, 3));
println @l string @r (list_at @l string @r (b, 2));

