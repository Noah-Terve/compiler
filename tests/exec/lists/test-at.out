[1, 2, 3, 4]
abc
1
4
3
